assets/javascripts/kitten/hoc/media-queries.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { createMatchMediaMax } from 'kitten/helpers/utils/media-queries'
+import {
+  SCREEN_SIZE_XS,
+  SCREEN_SIZE_M,
+} from 'kitten/constants/screen-config'
+
+export const mediaQueries = (WrappedComponent, hocProps = {}) => (
+  class extends Component {
+    constructor(props) {
+      super(props)
+
+      this.state = {
+        mqMobile: typeof(hocProps.mqMobile) !== 'undefined'
```

Comment:
Si ce sont des infos qui ne peuvent pas bouger durant le rendering du composant, je pense qu'on peut se passer de les mettre dans le `state`, non ?

Par exemple en le mettant dans `this.` ou même une fonction `mobileMediaQueryEnabled() { return typeof(hocProps.mqMobile) !== 'undefined' ? … : … }`.